content/post/076-collaborative-clusters.md

```diff
@@ -0,0 +1,59 @@
+---
+date: 2019-01-03
+url: 2020-01-03-collaborative-clusters
+title: Announcing collaborative clusters
+author: Hector Sanjuan
+---
+
+We are very excited to announce the first set of public *collaborative
+clusters* and the first iteration of the
+[collaborative clusters website](https://collab.ipfscluster.io).
+
+*Collaborative clusters* are an easy way to join and help improving distribution
+and data availability of specific datasets in the IPFS Network.
+
+Using IPFS Cluster's
```

Comment:
I feel like this is missing some sort of visual or way of communicating the value of a collaborative community cluster. Maybe not blocking here - but try opening a design shop request to @akrych or @ericronne to get their help landing this? Could be a nice visual to also include in the website going forward!

Reply:
Agreed, I will draft something